### PR TITLE
Upgrade uploadcare-rails version

### PR DESCRIPTION
**Pull request by kraft001**
## Description

Use a new version of uploadcare-rails to make the project compatible with ruby 3.1+

## Checklist

- [ ] Tests (if applicable)
- [ ] Documentation (if applicable)
- [ ] Changelog stub (or use [conventional commit messages](https://www.conventionalcommits.org/))

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -62,12 +62,12 @@ GEM
       zeitwerk (~> 2.3)
     addressable (2.8.1)
       public_suffix (>= 2.0.2, < 6.0)
-    api_struct (1.0.5)
-      dry-configurable
-      dry-inflector
-      dry-monads (~> 1)
-      hashie
-      http
+    api_struct (1.1.0)
```
Comment (rsedykh) [inline Gemfile.lock:65]: Shouldn't it be uploadcare-api_struct?

File: Gemfile.lock [hunk 2]
```diff
@@ -312,14 +312,14 @@ GEM
...
       rails (>= 4)
-      uploadcare-ruby (>= 4.2)
-    uploadcare-ruby (4.2.0)
-      api_struct (~> 1.0.1)
-      mimemagic
-      parallel
-      retries
+      uploadcare-ruby (>= 4.3)
+    uploadcare-ruby (4.3.0)
+      api_struct (= 1.1)
```
Comment (rsedykh) [inline Gemfile.lock:319]: Same question here. But I'm John Snow here and know nothing.

**Review by rsedykh — APPROVED** [review]
Check out a few minor questions.

I approve, but we need to find a way not to fails tests even though they pass in reality. :-(

File: Gemfile.lock (hunk 1 above)
Comment (kraft001) [reply Gemfile.lock]: Yes, exactly!

**Pull request merged**